camunda-archetype-ejb-war/src/main/resources/archetype-resources/README.md

```diff
@@ -57,7 +51,7 @@ and inspect it using
 [Camunda Cockpit](http://docs.camunda.org/latest/guides/user-guide/#cockpit).
 
 ${symbol_pound}${symbol_pound} Environment Restrictions
-Built and tested against Camunda Platform version ${camunda-version}.
+Built and tested against Camunda Platform version ${camunda-version} on WildFly 27 Application Server.
```

Comment:
🔧 Should we keep this more generic so we don't always have to adjust this in the future?
```suggestion
Built and tested against Camunda Platform version ${camunda-version} on WildFly 27+ Application Server.
```